bench/CMakeLists.txt

```diff
@@ -8,16 +8,36 @@ function(create_bench exfile croc)
   set(exname "bench-${exname}")
   create_ex_or_bench(${exfile} ${exname})
 
+  target_include_directories(${exname} PRIVATE ../examples)
   target_link_libraries(${exname} PRIVATE benchmark::benchmark)
   if(croc)
-    target_link_libraries(${exname} PUBLIC aligator::croc_compat croc_talos_arm_utils)
+    target_link_libraries(${exname} PRIVATE aligator::croc_compat croc_talos_arm_utils)
   endif(croc)
 endfunction()
 
+function(create_gar_bench exfile)
+  get_filename_component(exname ${exfile} NAME_WE)
+  set(exname "bench-${exname}")
+  add_executable(${exname} ${exfile})
+  message(STATUS "Adding cpp bench ${exname}")
+  set_target_properties(${exname} PROPERTIES LINKER_LANGUAGE CXX)
+  set_standard_output_directory(${exname})
+  target_include_directories(${exname} PRIVATE ${CMAKE_CURRENT_SOURCE_DIR})
+
+  target_link_libraries(${exname} PRIVATE aligator::gar benchmark::benchmark gar_test_utils)
+endfunction()
+
 create_bench("lqr.cpp" FALSE)
+create_bench("se2-car.cpp" FALSE)
 if(BUILD_CROCODDYL_COMPAT)
   create_bench("croc-talos-arm.cpp" TRUE)
   target_add_example_robot_data(bench-croc-talos-arm)
-  create_bench("multithreading.cpp" TRUE)
-  target_add_example_robot_data(bench-multithreading)
 endif()
+if(PINOCCHIO_V3)
```

Comment:
talos-walk benchmark requires BUILD_CROCODDYL_COMPAT, should be move under the corresponding if statement

Reply:
Excellent catch @LudovicDeMatteis